PensionManagementUserLoginService/Models/Context/AuthDbContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace PensionManagementUserLoginService.Models.Context
+{
+    public class AuthDbContext: IdentityDbContext
+    {
+        public AuthDbContext(DbContextOptions<AuthDbContext> options) : base(options)
+        {
+
+        }
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            var readerRoleId = "92327fd8-20fc-42d8-9bd8-536a746bc3ec";
+            var writerRoleId = "ab185359-929b-48ef-85b3-6acb0637dcfd";
```

Comment:
same here is this the default if or you have provided it

Reply:
we have generated new Guid Id 
where it is a unique identifier for the writerroleId